src/pages/write-smart-contracts/hello-world-tutorial.md

```diff
@@ -12,9 +12,15 @@ images:
 
 ## Introduction
 
-In the world of smart contracts, everything is a blockchain transaction. You use tokens in your wallet to deploy a smart contract in a transaction, and each call to that contract after it's published is a transaction, too. That means that at each step, tokens are being exchanged as transaction fees. This tutorial introduces you to this mode of programming, which transforms blockchains into powerful state machines capable of executing complex logic.
+In the world of smart contracts, everything is a blockchain transaction. You use tokens in your wallet to deploy a smart
+contract in a transaction, and each call to that contract after it's published is a transaction, too. That means that at
```

Comment:
So I had intended to bring this up in my assessment of the docs, but it would be nice to enforce a max line length in the markdown (here I'm using 120 characters). The upside with multiple lines is that git diffs are much cleaner, the downside is that you do have to reflow content when editing it.

If you think this is outside the score of this PR I can revert.